spec/dates_from_string_spec.rb

```diff
@@ -496,6 +496,62 @@
       expect(subject.get_structure(input)).to eq(output)
     end
 
+    it 'finds full date year month and day in USA comma-delimited format' do
+      input = 'February 2, 2015'
+      output = [
+        {:type=>:month, :value=>"02", :distance=>1, :key_words=>[]},
+        {:type=>:day, :value=>"2", :distance=>1, :key_words=>[]},
+        {:type=>:year, :value=>"2015", :distance=>0, :key_words=>[]}
+      ]
+
+      expect(subject.get_structure(input)).to eq(output)
+    end
+
+    it 'finds day when sepcified with a single digit' do
+      input = '2 Feb in the year 2015'
+      output = [
+        {:type=>:day, :value=>"2", :distance=>1, :key_words=>[]},
+        {:type=>:month, :value=>"02", :distance=>4, :key_words=>[]},
+        {:type=>:year, :value=>"2015", :distance=>0, :key_words=>[]}
+      ]
+
+      expect(subject.get_structure(input)).to eq(output)
+    end
+
+    it 'finds date when abbreviated month buttressed by comma' do
+      input = '2 Feb, 2015'
+      output = [
+        {:type=>:day, :value=>"2", :distance=>1, :key_words=>[]},
+        {:type=>:month, :value=>"02", :distance=>1, :key_words=>[]},
+        {:type=>:year, :value=>"2015", :distance=>0, :key_words=>[]}
+      ]
+
+      expect(subject.get_structure(input)).to eq(output)
+    end
+
+    it 'finds date when month buttressed by comma' do
+      input = '2 February, 2015'
+      output = [
+        {:type=>:day, :value=>"2", :distance=>1, :key_words=>[]},
+        {:type=>:month, :value=>"02", :distance=>1, :key_words=>[]},
+        {:type=>:year, :value=>"2015", :distance=>0, :key_words=>[]}
+      ]
+
+      expect(subject.get_structure(input)).to eq(output)
+    end
+
+    it 'finds day with a specified "nd" ordinal' do
```

Comment:
Why you don't want to add a test for ['st', 'th'] cases?

Reply:
Good question! I'll see if I can  get around to updating the the PR soon.

Reply:
Ok, after it I merge and bump new version